packages/stark-ui/src/modules/breadcrumb/components/breadcrumb.component.ts

```diff
@@ -79,33 +81,31 @@ export class StarkBreadcrumbComponent extends AbstractStarkUiComponent implement
 	 * Component lifecycle hook
 	 */
 	public ngOnDestroy(): void {
-		if (this.transitionHookDeregisterFn) {
-			this.transitionHookDeregisterFn();
-		}
+		this.deregisterTransitionHook();
+		this.deregisterTransitionHook();
```

Comment:
This line is duplicated

Reply:
🙈 Also the reason why the build is failing I think.